app/src/main/res/layout/dashboard_tools_spotlight_list_item.xml

```diff
@@ -0,0 +1,105 @@
+        tools:layout_height="240dp">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layoutDirection="@{primaryTranslation.languageCode != null ? TextUtils.getLayoutDirectionFromLocale(primaryTranslation.languageCode) : View.LAYOUT_DIRECTION_INHERIT}">
+
+
+            <org.cru.godtools.base.ui.view.DaggerPicassoImageView
+                android:id="@+id/toolsImage"
+                style="@style/Widget.GodTools.Tools.Card.Banner"
+                android:layout_width="match_parent"
+                android:layout_height="161dp"
+                android:scaleType="centerCrop"
+                android:src="@{banner}"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                tools:src="@tools:sample/backgrounds/scenic" />
+
+            <ProgressBar
+                style="@style/Widget.GodTools.Tools.Download.ProgressBar"
+                android:layout_width="match_parent"
+                android:layout_height="10dp"
+                android:progress="@{downloadProgress}"
+                android:visibility="gone"
+                app:layout_constraintBottom_toBottomOf="@id/toolsImage"
+                tools:max="12"
+                tools:progress="5"
+                tools:visibility="visible" />
+
+            <com.google.android.material.imageview.ShapeableImageView
+                style="@style/Widget.GodTools.Tools.Card.Action.Favorite"
+                android:layout_marginTop="11dp"
+                android:layout_marginEnd="11dp"
+                android:onClick="@{() -> tool.isAdded ? callbacks.removeTool(tool, primaryTranslation ?? parallelTranslation) : callbacks.addTool(tool.code)}"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                app:selected="@{tool.isAdded}"
+                app:srcCompat="@drawable/ic_favorite" />
+
+            <TextView
+                android:id="@+id/toolName"
+                style="@style/TextAppearance.GodTools.Tools.Card.Title"
+                android:layout_width="wrap_content"
+                android:layout_height="0dp"
+                android:layout_marginStart="16dp"
+                android:layout_marginTop="10dp"
+                android:lines="2"
+                android:text="@{tool.name}"
+                app:layout_constraintBottom_toTopOf="@id/parallelLanguageText"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/toolsImage"
+                tools:text="Tool Name" />
+
+            <androidx.appcompat.widget.AppCompatTextView
+                android:id="@+id/parallelLanguageText"
+                style="@style/Widget.GodTools.Tools.Card.Subtitle"
```

Comment:
actually looking at this again, there is a parallel language style that is the correct one to use here. my mistake in originally suggesting the subtitle one.